### PR TITLE
attack_verb is now always present tense

### LINKED ISSUE #9210: Slamsed V2. BIBLE EDITION
#### Description of issue

Bible hitting on head says slammeds wtf whoever fixed this needs to die
#### Difference between expected and actual behavior

It should say slammed, wtf
#### Steps to reproduce

Hit someone on head with bible until it says slammeds
#### Specific information for locating

Bible, hitting someone on the head says SLAMMEDS
WHO FIXED THIS I WANT THEIR HEAD ON A STAKE
#### Server revision

bfd48c1a99c2db24a9596b9931c7002e3d9bf5dc
#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [X] Issue could be reproduced at least once
- [X] Issue could be reproduced by different players
- [X] Issue could be reproduced in multiple rounds
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by Intigracy**
also removes an unused and unticked hydro.dm which has the old defines for plants n shit

fixes #9210

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/constructable/swords.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	force = 15
 	throwforce = 10
 	sharpness = 1.2
-	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	attack_verb = list("attacks", "slashs", "stabs", "slices", "tears", "rips", "dices", "cuts")
```
Comment (Exxion) [inline code/modules/projectiles/guns/projectile/constructable/swords.dm]: > slashs

Comment (Intigracy) [inline code/modules/projectiles/guns/projectile/constructable/swords.dm]: ty

File: vgstation13.dme [hunk 2]
```diff
@@ -1069,6 +1068,7 @@
 #include "code\modules\events\link_with_centcomm.dm"
 #include "code\modules\events\masshallucination.dm"
 #include "code\modules\events\meteors.dm"
+#include "code\modules\events\mob_swarm.dm"
```
Comment (Intigracy) [inline vgstation13.dme]: idk why I included the dme but w/e, it's putting it in order

**Comment by Intigracy** [comment]
This is done unless anyone else sees another typo.

I'd uninclude the dme but it's not even showing in my changed files.

**Comment by Intigracy** [comment]
Confrictu soon with golem race PR.

**Pull request merged**